src/ngraph/runtime/cpu/pass/cpu_rnn_fusion.cpp

```diff
@@ -540,8 +541,12 @@ void ngraph::runtime::cpu::pass::RNNFusion::construct_rnn_lstm_fprop()
                                                    ref_rnn_type);
     auto lstm_goe = std::make_shared<ngraph::op::GetOutputElement>(lstm, 1);
     // We cannot attach labels to multi-output nodes, so we attach a label to the goe instead
-    auto lstm_goe_label =
-        std::make_shared<pattern::op::Label>(lstm_goe, nullptr, NodeVector{lstm_goe});
+    auto lstm_goe_label = std::make_shared<pattern::op::Label>(
+        lstm_goe,
+        nullptr,
+        OutputVector{std::make_shared<pattern::op::Or>(
```

Comment:
Why do we need Or here? Aren't we always slice lstm_goe?

Reply:
The last cell is connected to the graph by a different output than the output used for cell to cell connections.